sys/auto_init/netif/auto_init_at86rf2xx.c

```diff
@@ -48,6 +51,12 @@ void auto_init_at86rf2xx(void)
         LOG_DEBUG("[auto_init_netif] initializing at86rf2xx #%u\n", i);
 
         at86rf2xx_setup(&at86rf2xx_devs[i], &at86rf2xx_params[i]);
+#ifdef MODULE_GNRC_GOMACH
+        gnrc_netif_gomach_create(_at86rf2xx_stacks[i],
+                                 AT86RF2XX_MAC_STACKSIZE,
+                                 AT86RF2XX_MAC_PRIO, "at86rf2xx-gomach",
+                                 (netdev_t *)&at86rf2xx_devs[i]);
+#else
 #ifdef MODULE_GNRC_LWMAC
```

Comment:
use `#elif defined(MODULE_GNRC_LWMAC)` instead and remove the extra `#endif` below. Optionally, to keep everything nice and unified, use `#if defined(MODULE_GNRC_GOMACH)` above as well.